tests/Unit/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/Test_WrappedGr.cpp

```diff
@@ -44,7 +44,7 @@ void compare_different_wrapped_solutions(const double mass,
 template <typename SolutionType>
 void test_copy_and_move(const SolutionType& solution) {
   test_copy_semantics(solution);
-  auto solution_copy = solution;
+  const auto& solution_copy = solution;
```

Comment:
this is just a reference now. Can you just pass in `solution` directly?

Reply:
Done.